apps/artemis-notification/src/main.ts

```diff
@@ -0,0 +1,210 @@
+import {
+    ApolloClient,
+    createHttpLink, gql,
+    InMemoryCache
+} from '@apollo/client';
+import { setContext } from '@apollo/client/link/context';
+import { MongoClient, Db } from 'mongodb';
+import fetch from 'node-fetch';
+import {
+    setInterval
+} from 'timers/promises';
+import * as https from 'https';
+import * as jwt from 'jsonwebtoken';
+import * as admin from 'firebase-admin';
+import { nanoid } from 'nanoid';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+const URI = `mongodb://${process.env.MONGODB_USER}:${process.env.MONGODB_PASS}@${process.env.MONGODB_HOST}:${process.env.MONGODB_PORT}`;
+
+const forgeToken = async (): Promise<string> => {
+    const client = new MongoClient(URI, {});
+    try {
+        await client.connect();
+        const db = client.db(process.env.MONGODB_NAME);
+        const user = await db.collection('users').findOne({ email: process.env.DEFAULT_EMAIL });
+        const token = jwt.sign(
+            {
+                'https://hasura.io/jwt/claims': {
+                    'x-hasura-allowed-roles': [user.role],
+                    'x-hasura-default-role': user.role,
+                    'x-hasura-user-id': user._id,
+                },
+                user: user,
+            },
+            process.env.JWT_SECRET
+        );
+        return token;
+    } catch (error) {
+        console.error(error);
+    }
+};
+
+const fetchHijackUpdates = async () => {
+    const jwt: string = await forgeToken();
+
+    const agent = new https.Agent({
+        rejectUnauthorized: false,
+    })
+    const authLink = setContext(async (_, { headers }) => {
+        return {
+            headers: {
+                ...headers,
+                authorization: jwt ? `Bearer ${jwt}` : '',
+            },
+        };
+    })
+    const httpLink =
+        createHttpLink({
+            uri: `https://localhost/api/graphql`,
+            fetch,
+            fetchOptions: {
+                agent: agent
+            },
+            useGETForQueries: false,
+        });
+
+
+    const client = new ApolloClient({
+        link: authLink.concat(httpLink),
+        cache: new InMemoryCache(),
+    });
+
+    for await (const startTime of setInterval(10000, Date.now())) {
+        const date = new Date();
+        console.log((date.getTime() - startTime) / 10000);
```

Comment:
`startTime` is not used anywhere else we probably do not need it.
Perhaps is there a way to refactor `for await (const startTime of setInterval(10000, Date.now())) {`?